### PR TITLE
Adds NLP with PyTorch basic example to README

### PR DESCRIPTION
**Pull request by joshdevins**
The Machine Learning section now has two sub-sections — one for
traditional regression/classification and the other for NLP with
PyTorch. The examples show two ways to upload models from the Hugging
Face model hub.

### CODE REVIEW

**Review by sethmlarson — APPROVED** [review]
Documentation looks good, one thought:

File: README.md [hunk 1]
```diff
@@ -215,3 +217,43 @@ to be serialized and used as an inference model in Elasticsearch
...
+```
+
+```python
+>>> import elasticsearch
+>>> from pathlib import Path
+>>> from eland.ml.pytorch import PyTorchModel
+>>> from eland.ml.pytorch.transformers import TransformerModel
+
+# Load a Hugging Face transformers model directly from the model hub
+>>> tm = TransformerModel("elastic/distilbert-base-cased-finetuned-conll03-english", "ner")
```
[the remark this answers is not in the document]
Comment (joshdevins) [reply README.md:242]: There's currently two methods on `TransformerModel`. One is to download and save locally in the right format that we need and the second is to get the model ID that is Elasticsearch compatible. I agree that it's not totally logical to have an object here, but I fear one mega function that returns 4 values (existing 3+ES model ID?).

[the remark this answers is not in the document]
Comment (sethmlarson) [reply README.md:242]: `TransformerModel` could be a dataclass with `model_path`, `config_path`, `vocab_path`, and `es_model_id`? Then `TransformerModel.download_model()` could essentially be what `TransformerModel.__init__().save(path)` is today (but return the `TransformerModel` instead).

Maybe I can open a separate PR with this suggestion and you can see how it'd look+feel?

[the remark this answers is not in the document]
Comment (joshdevins) [reply README.md:242]: Yeah, happy to iterate in another PR

**Comment by joshdevins** [comment]
@sethmlarson seems I can't merge because of the tests that still fail on `main`. Can you override that?

**Pull request merged**